catalog/message/catalog.json

```diff
@@ -36,7 +36,7 @@
       "odrl:hasPolicy": [
         {
           "@type": "odrl:Offer",
-          "dspace:providerId": "http://example.com/Provider",
+          "odrl:assigner": "http://example.com/Provider",
```

Comment:
Please only use "dspace:providerId" instead of "odrl:assigner" and "dspace:consumerId" instead of "odrl:assignee" , also in all other examples.